### DISCUSSION TITLE
Welcome to RazorLight Discussions!

### DISCUSSION THREAD

**Discussion by toddams** [toddams/RazorLight#405] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jzabroski** [comment]
@toddams Can I get promoted to maintainer so I can change a few things, like move from Travis CI to GitHub Actions, as well as give others contributor access?

**Reply by jzabroski** [reply]
Context: Travis-ci.org is shutting down, everything is moving to travis-ci.com

![image](https://user-images.githubusercontent.com/447485/102827684-08275580-43b1-11eb-8f4d-37f156a1574c.png)
